src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -567,6 +567,45 @@ def _update_mgrs(self, num, hosts=None):
         orchestrator.raise_if_exception(completion)
         return HandleCommandResult(stdout=completion.result_str())
 
+    @_write_cli('orchestrator mgr add',
+                "name=hosts,type=CephString,n=N,req=true",
```

Comment:
i'm not sure req is required here, since in principle we could wire this and the mon add/rm up to rook too (and have it +1 or -1 the deployment size)?

Reply:
I thought rook will still use `mon/mgr update`. The intended usage for `mon/mgr add/rm` is for ssh-orch users imo.

Reply:
If we have two different apis for different orchestrators, we basically failed to make a proper api to unify all the backends. And thus make it harder for users like the dashboard to build a client.

Reply:
Mh, I get the point. I fear we can't unify the interface without dropping either implementation though. Either rook adapts to {add, rm} or the ssh-orch drops it and re-uses the not-so-userfriendly {update}.
Either solution doesn't sound too great to me..

Reply:
@bk201 any input from you?

Reply:
Either way would work for the dashboard. But we should only keep one.

I tried to map the situation to restful, `add/remove` methods are much instinctive when adding/removing resources from UI/CLI. And `update` would likely to be used for updating attributes inside resources.